code/obj/item/organ.dm

```diff
@@ -1296,6 +1296,25 @@
 		else
 			return ..()
 
+/obj/item/skull/attack_self(mob/user as mob)

```

Comment:
I would have an interaction limiter so people can't spam this constantly. Might be built into items, but I doubt it.

Reply:
Added a spamcheck